utils/build-script-impl

```diff
@@ -3069,13 +3093,16 @@ for host in "${ALL_HOSTS[@]}"; do
             #
             # Exclude shell scripts and static archives.
             # Exclude swift-api-digester dSYM to reduce debug toolchain size.
-            # Run sequentially -- dsymutil is multithreaded and can be memory intensive
+            # Tweak carefully the amount of parallelism -- dsymutil can be memory intensive and
+            # as such too many instance can exhaust the memory and slow down/panic the machine
+            printJSONStartTimestamp dsymutil
```

Comment:
I recognize this is not strictly required to add the parameter, but it seemed to make sense given we are now able to tune this in a more direct way